rmf_demos_tasks/rmf_demos_tasks/teleop_action.py

```diff
@@ -44,19 +44,19 @@ def __init__(self, argv=sys.argv):
 
         self.pub = self.create_publisher(
           PathRequest, 'robot_path_requests', 1)
-        pts = [[5.3,-4.9], [3.3, -8.9], [5.3, -9.9], [7.3, -6.1], [5.3,-4.9]]
+        pts = [[5.3, -4.9], [3.3, -8.9], [5.3, -9.9], [7.3, -6.1], [5.3, -4.9]]
```

Comment:
Should the path be configurable as a list?

Reply:
Sounds right, updated and the script now takes in path coordinates and map name as arguments